index.html

```diff
@@ -17,16 +17,21 @@
     <header>
         <nav class="container-nav">
             <a href="#" class="logo-name">Federica Ulzurrun</a>
-            <div class="navlog">
-                <img src="img/navhamburger.png" alt="20px">
-            </div>
-            <ul class="nav-desk">
-                <li class="page-parts">portfolio</li>
-                <li class="page-parts">about</li>
-                <li class="page-parts">contact</li>
+            <ul class="nav-menu">
+                <li class="page-parts"><a class="nav-link" href="#portfolio">portfolio</a></li>
+                <li class="page-parts"><a href="#about" class="nav-link">about</a></li>
+                <li class="page-parts"><a href="#cont" class="nav-link">contact</a></li>
```

Comment:
- [ ] As pointed out by the previous reviewers, please kindly ensure that the hamburger disappears in the desktop view while the nav menu shows up. At the moment, the hamburger still comes up in the desktop view. Please see the below image for reference.

![image](https://user-images.githubusercontent.com/43995830/223467368-53958d16-047f-462e-87c6-cb35a4541b5b.png)

Reply:
@AnselemOdims hi!! I have changed but my page it’s responsive until max-width 1440px! I have to made it responsive for even more pixels?
Because until that width it’s working perfect